src/java/htsjdk/samtools/metrics/MetricsFile.java

```diff
@@ -84,6 +84,13 @@ public void addAllMetrics(final Iterable<BEAN> beanz) {
         else return null;
     }
 
+    /** Returns the histogram contained in the metrics file if any. Adds capability of retrieving multiple histograms
+     *  from metrics file. */
+    public Histogram<HKEY> getHistogram(final int index) {
+        try { return this.histograms.get(index); }
+        catch (final ArrayIndexOutOfBoundsException excep) { throw new SAMException("Index is out of bounds: " + index); }
```

Comment:
Oh... right, there's an existing method that returns nulls.. That makes sense why you were doing it that way.  I wasn't thinking about that.